### ISSUE TITLE
Engage with FNP committee for governance of the FNP catalogue

### ISSUE THREAD

**Issue by Metaduck** [idn-au/.github#66]
27-Feb Next meeting is in April. Not a priority at the moment.

Project update has been provided but no further word from them.

Because they have a University wide mandate, we are trying to bring together ALL the groups to cover ID governance

===============
Committee requested information from the FNP (general overview) which is a pro-forma tbc

Meeting Oct-31 for FNP and will report back on any updates

23-jan-24 Committee to resume in February. can follow this up after they have resumed.

**Comment by Metaduck** [comment]
Meetings have been set up to progress this

**Comment by Metaduck** [comment]
Still waiting on a meeting to be set up - hopefully in November.

**Comment by moniqueharris** [comment]
FNP Committee no longer exists. Will need to engage with another group to finalise this piece. To be determined.

**Comment by moniqueharris** [comment]
Meeting set for 21/07

**Comment by moniqueharris** [comment]
Meeting with Peter is being rescheduled. Awaiting new date.
